x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/privilege/IndexPrivilege.java

```diff
@@ -177,7 +177,7 @@ public final class IndexPrivilege extends Privilege {
     public static final IndexPrivilege MANAGE_FOLLOW_INDEX = new IndexPrivilege("manage_follow_index", MANAGE_FOLLOW_INDEX_AUTOMATON);
     public static final IndexPrivilege MANAGE_LEADER_INDEX = new IndexPrivilege("manage_leader_index", MANAGE_LEADER_INDEX_AUTOMATON);
     public static final IndexPrivilege MANAGE_ILM = new IndexPrivilege("manage_ilm", MANAGE_ILM_AUTOMATON);
-    public static final IndexPrivilege MANAGE_DLM = new IndexPrivilege("manage_dlm", MANAGE_DLM_AUTOMATON);
+    public static final IndexPrivilege MANAGE_LIFECYCLE = new IndexPrivilege("manage_lifecycle", MANAGE_LIFECYCLE_AUTOMATON);
```

Comment:
I'm wondering if `manage_data_stream_lifecycle` or `manage_data_lifecycle` are better names, as they disambiguates between this and ILM. Not a strong preference though!

Reply:
I was on the fence about this too. It looks short and nice when it's `manage_lifecycle` but indeed it's not as specific. I will change it to `manage_data_stream_lifecycle`. This is the most accurate name right now.